context/PokemonContextUtils.tsx

```diff
@@ -0,0 +1,12 @@
+import { Pokemon } from '../api/pokemon';
```

Comment:
if you create file with only types, it's better to call it `types.tsx` or something like that rather than utils. I'd also keep it lowercase and use uppercase only for components (so e.g. `pokemonContextTypes.tsx`)